### PR TITLE
(0.9.1) Move testing and docs to buildkite (+ minor GPU bug fix)

### PR DESCRIPTION
**Pull request by jagoosw**

### CODE REVIEW

**Review by navidcy — APPROVED** [review]
noice!

**Comment by jagoosw** [comment]
I think I'll also move CPU testing and docs building to buildkite since it seems to be so much faster than GitHub actions

**Comment by navidcy** [comment]
Definitely. Go for it!

**Comment by navidcy** [comment]
Where do we set up the buildkite??

**Comment by navidcy** [comment]
if we'll have both CPU and GPU tests on buildkite then let's not call the buildkite pipeline `gpu-tests`

**Comment by jagoosw** [comment]
Yeah I've changed the name now. 

I'm not sure how oceananigans has it in the repo but I'm setting it up here https://buildkite.com/oceanbiome-dot-jl/oceanbiome. I'll add you to the org.

**Comment by jagoosw** [comment]
> Where do we set up the buildkite??

Also if you meant what is the host, we've now got a computer with an GPU that we can host the buildkite agent on.

**Comment by navidcy** [comment]
> > Where do we set up the buildkite??
> 
> Also if you meant what is the host, we've now got a computer with an GPU that we can host the buildkite agent on.

I was referring to any yml file we use to set up the buildkite pipeline?

**Comment by jagoosw** [comment]
I think you should be able to edit it here https://buildkite.com/oceanbiome-dot-jl/oceanbiome/settings/steps

**Comment by jagoosw** [comment]
Ah I just realised you can put it in the repo so I'll move it now

File: .buildkite/pipeline.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+env:
+  JULIA_VERSION: "1.9.3"
```
Comment (navidcy) [inline .buildkite/pipeline.yml:2]:
```suggestion
  JULIA_VERSION: "1.9.4"
```

**Comment by jagoosw** [comment]
The duplication of the status' in the list should be fixed on the next commit, I don't think that it is reading the pipeline file in the repo and I can't find any docs suggesting how you can do that. Maybe someone who setup the Oceananigans testing would know?

**Comment by navidcy** [comment]
Let's ensure that Documenter has been set using appropriate ssh keys and what not so that it has rights to push before we merge.

**Comment by jagoosw** [comment]
Yeah definitely, I think it should be right now so hopefully it'll push a preview when it's done this time

**Comment by navidcy** [comment]
https://buildkite.com/oceanbiome-dot-jl/oceanbiome/builds/45#018bd01c-23f7-4fb7-a411-83b6e3bde508/21-2823

> Let's ensure that Documenter has been set using appropriate ssh keys and what not so that it has rights to push before we merge.

https://buildkite.com/oceanbiome-dot-jl/oceanbiome/builds/45#018bd01c-23f7-4fb7-a411-83b6e3bde508/21-2823

**Comment by jagoosw** [comment]
I think I've worked it out now, I had to make a new environmental variable on the agent I think

**Comment by navidcy** [comment]
Seems that it worked!

**Review by navidcy — APPROVED** [review]

**Pull request merged**